ext/RunProcess.php

```diff
@@ -58,7 +58,7 @@
 class RunProcess extends Extension
 {
     /**
-     * @var array<string, int>
+     * @var array<int|string, mixed>
```

Comment:
When would it make sense to have integer keys in configuration?

Reply:
This might be unique to the `RunProcess` extension, but take the following config:

```yaml
extensions:
  enabled:
    - Codeception\Extension\RunProcess:
        0: 'chromedriver --url-base=/wd/hub'
        sleep: 1
```

Which is parsed as:

```php
array (
  'sleep' => 1,
  0 => 'chromedriver --url-base=/wd/hub',
)
```

Hence why `int|string` key type is required for the `config` property (on `RunExtension`).